### PR TITLE
Logs tool usage

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Thanks to https://github.com/tgstation/tgstation/pull/64375 and https://github.com/tgstation/tgstation/pull/64428 - Tool logging seems like it can be added as a reliable thing.

## Why It's Good For The Game

I find it potentially useful for staff, there's certain scenarios you would want to figure out who did what to a certain machine
Examples are:
- Welding doors shut
- Who deconstructed an ORM
- Who wrenched vendors to block doors
- Who deconstructed all the techfabs
- Ect

I would mention welding walls but that's logged in game (which could probably be removed now)

image

## Changelog

:cl:
admin: Tool usage is now (config-locked) logged!
/:cl:

### CODE REVIEW

File: code/game/world.dm [hunk 1]
```diff
@@ -127,6 +127,7 @@ GLOBAL_VAR(restart_counter)
 
 	GLOB.world_game_log = "[GLOB.log_directory]/game.log"
 	GLOB.world_silicon_log = "[GLOB.log_directory]/silicon.log"
+	GLOB.world_silicon_log = "[GLOB.log_directory]/tools.log"
```
Comment (Mothblocks) [inline code/game/world.dm:130]: silicon log? Bro

Comment (JohnFulpWillard) [reply code/game/world.dm:130]: oops

File: code/controllers/configuration/entries/general.dm [hunk 2]
```diff
@@ -70,6 +70,8 @@
 /datum/config_entry/flag/log_law/DeprecationUpdate(value)
 	return value
 
+/datum/config_entry/flag/log_tools // log using tools on machines
```
Comment (Mothblocks) [inline code/controllers/configuration/entries/general.dm:73]:
```suggestion
/// Log usage of tools
/datum/config_entry/flag/log_tools
```

1. Real autodoc
2. It has no reason to stay machine specific (if it even is now)

**Comment by maxspells** [comment]
speedmerge

**Comment by Mothblocks** [comment]
Also logs have always been compressed they're just now double super duper compressed now

**Comment by JohnFulpWillard** [comment]
https://github.com/tgstation/tgstation/pull/64428/files removes screwdriver/wirecutter from airlocks, but not welders. Once that PR is merged, I'll do it so welding airlocks is logged in tool rather than game, I just don't know if it should be a separate PR.

**Comment by Timberpoes** [comment]
> Also logs have always been NTFS compressed they're just now double super duper compressed now

Allowing for EVEN MORE LOGGING. 

With such great power comes an equally great responsibility. To log all the things!

**Pull request merged**